### PR TITLE
Code coverage removal

### LINKED ISSUE #711: Remove Codecov integration
Not used anymore. We use SonarCloud for showing the coverage

### PR DESCRIPTION
**Pull request by bakgerman**
Resolves #711

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -29,8 +29,6 @@ test_script:
   - OpenCover\tools\OpenCover.Console.exe -register:user -returntargetcode -target:"%xunit20%\xunit.console.x86.exe" -targetargs:"\"c:\projects\nlogweb\tests\NLog.Web.Tests\bin\Release\net452\NLog.Web.Tests.dll\" -appveyor -noshadow"  -returntargetcode -filter:"+[NLog.Web]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -oldstyle -output:coverage.xml
```
Comment (snakefoot) [inline appveyor.yml:29]: Maybe also remove the use of `OpenCover.Console.exe` and `nuget.exe install OpenCover` (just above)

Comment (bakgerman) [reply appveyor.yml]: completed

Comment (bakgerman) [reply appveyor.yml]: i have done that, the build seems to now test all .NET platforms.

**Comment by snakefoot** [comment]
AppVeyor build looks good now, and all checks are green. Thank you for the help with the cleanup.

**Pull request merged**